refaster-support/src/main/java/tech/picnic/errorprone/refaster/AnnotatedCompositeCodeTransformer.java

```diff
@@ -78,16 +78,16 @@ public final void apply(TreePath path, Context context, DescriptionListener list
     }
   }
 
-  @SuppressWarnings("RestrictedApiChecker" /* We create a heavily customized `Description` here. */)
+  @SuppressWarnings("RestrictedApi" /* We create a heavily customized `Description` here. */)
   private Description augmentDescription(
       Description description, CodeTransformer delegate, Context context) {
     String shortCheckName = getShortCheckName(description.checkName);
     return Description.builder(
             description.position,
             shortCheckName,
             getLinkPattern(delegate, shortCheckName).orElse(null),
-            overrideSeverity(getSeverity(delegate), context),
             getDescription(delegate))
+        .overrideSeverity(overrideSeverity(getSeverity(delegate), context))
```

Comment:
PItest warns about this call not being covered, but that's because the tests are in another module. If I drop this line then `RefasterTest` does fail.